_posts/2019-07-08-recappin-post.md

```diff
@@ -63,11 +63,11 @@ Imagine an adventure party voluntarily formed of the bravest souls (cough*includ
 
 #### _2. Talks given by the attendees_
 
-Those in the audience who have had their talks selected for DDDEM rose to the occasion and presented a quickly prep-ed 3 mins talks on a topic of their liking to the whole room. The rest of us then anonymously wrote down 1 thing we liked and 1 thing that could’ve been better on two post-its, for each of the speaker’s talk. Tell you what though, I really struggled finding the thing to improve for so many of them, they did so well!
+Those in the audience who have had their talks selected for DDDEM rose to the occasion and presented a quickly prep-ed 3 mins talk on a topic of their liking to the whole room. The rest of us then anonymously wrote down 1 thing we liked and 1 thing that could’ve been better on two post-its, for each of the speaker’s talk. Tell you what though, I really struggled finding the thing to improve for so many of them, they did so well!
```

Comment:
I think was correctly pluralised

Reply:
Thanks for reviewing and no worries, will change back to plural 'talks'. Should we remove the 'a' in that case though? So it will read '`...and presented quickly prep-ed 3 mins talks...`' rather than '`...and presented a quickly prep-ed 3 mins talk...`'

Reply:
Yeah, that sounds good. Thanks :)